fs/fs_zenfs.cc

```diff
@@ -574,6 +574,32 @@ IOStatus ZenFS::DeleteFile(const std::string& fname, const IOOptions& options,
   return s;
 }
 
+IOStatus ZenFS::GetFileCreationTime(const std::string& f, const IOOptions& options,
```

Comment:
Implement GetFileModificationTime that is part of the file system api in stead, and use that in the utility

Reply:
Ok.